Tools/autotest/locations.txt

```diff
@@ -57,3 +57,4 @@ Kawachi=35.879129,140.339683,7,0
 SpringValley= -35.280252,149.005821,597.3,5
 Pyramid=29.9764,31.1339,0,0
 AAMEastField=39.842288,-105.212928,1809,106
+Almere=52.365021, 5.201260, 0, 270
```

Comment:
Does this actually work for you?  I didn't think the parser was actually lenient enough to allow spaces in there!

Note that you can *also* have a local set of flying sites:

```
pbarker@bluebottle:~$ cat /home/pbarker/.config/ardupilot/locations.txt
GSP=-35.207897,149.043640,582.840027,270
GSPM=-35.207745,149.043099,582.840027,270
MTRG=-35.2094851,149.0571052,595.800,90 # mount roger's school oval
MURN=-35.211195,149.040381,574.2,45 # mount roger's school oval
COBW=-35.620633,144.127053,247,135 # Oval on Cobwell Road
pbarker@bluebottle:~$ 
```

I'm not sure what the Windows equivalent of that is, sorry!